kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.stream.array;
+
+import java.util.OptionalDouble;
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+
+    static double getAverage(int[] numbers) {
+        IntStream.range(0, numbers.length)
+                .forEach(System.out::println);
+        double averageNumber = IntStream.range(0, numbers.length)
```

Comment:
Ta funckcja nie liczy sredniej inputu, tylko srednia z `(0, numbers.length)`. To nie zadziala w tescie dla np `{0,5,15}`